### PR TITLE
add java release highlights May 2021

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

**Review by czubair — APPROVED** [review]

File: releases/2021-05/index.md [hunk 1]
```diff
@@ -11,9 +11,17 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
...
-*
-*
-*
+- Azure Cosmos DB client library for Java is releasing a new GA.
+    - Adds `backendLatencyInMs` in `CosmosDiagnostics` for `DIRECT` connection mode and `retryContext` in `CosmosDiagnostics` for query operations.
+- Java Azure Core library is releasing a new GA.
+    - Adds support for Challenge Based Authentication in BearerTokenAuthenticationPolicy.
+    - Updated logic to eagerly read response bodies to include return types void and Void.
+- Java Azure Messaging EventGrid library is releasing a new GA.
+    - Adds new Storage system events `StorageAsyncOperationInitiatedEventData`,  `StorageBlobTierChangedEventData` and new Policy Insights system events `PolicyInsightsPolicyStateCreatedEventData`, `PolicyInsightsPolicyStateChangedEventData`,  `PolicyInsightsPolicyStateDeletedEventData`.
```
Comment (czubair) [inline releases/2021-05/index.md:20]:
```suggestion
    - Adds new Storage system events `StorageAsyncOperationInitiatedEventData`,  `StorageBlobTierChangedEventData` and new Policy Insights system events: `PolicyInsightsPolicyStateCreatedEventData`, `PolicyInsightsPolicyStateChangedEventData`,  `PolicyInsightsPolicyStateDeletedEventData`.
```

Comment (czubair) [inline releases/2021-05/index.md:20]:
```suggestion
    - Adds new Storage system events: `StorageAsyncOperationInitiatedEventData`,  `StorageBlobTierChangedEventData` and new Policy Insights system events `PolicyInsightsPolicyStateCreatedEventData`, `PolicyInsightsPolicyStateChangedEventData`,  `PolicyInsightsPolicyStateDeletedEventData`.
```

File: releases/2021-05/index.md [hunk 2]
```diff
@@ -11,9 +11,17 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
...
 
-*
-*
-*
+- Azure Cosmos DB client library for Java is releasing a new GA.
+    - Adds `backendLatencyInMs` in `CosmosDiagnostics` for `DIRECT` connection mode and `retryContext` in `CosmosDiagnostics` for query operations.
+- Java Azure Core library is releasing a new GA.
+    - Adds support for Challenge Based Authentication in BearerTokenAuthenticationPolicy.
+    - Updated logic to eagerly read response bodies to include return types void and Void.
+- Java Azure Messaging EventGrid library is releasing a new GA.
```
Comment (czubair) [inline releases/2021-05/index.md:19]: Event Grid's name does not have Azure in front of it: https://azure.microsoft.com/en-us/services/?query=event+grid

**Comment by weshaggard** [comment]
This is old and not needed any more so closing.

**Pull request closed without merging**